src/sensors/humidity.rs

```diff
@@ -0,0 +1,37 @@
+//! Module providing humidity sensor functionality.
+
+use super::{LocalisedSensorMetadata, SensorTemplate, SensorTemplateError, Sensors};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+pub struct HumiditySensor {
+    #[serde(flatten)]
+    pub metadata: LocalisedSensorMetadata,
+    pub unit: String,
+    pub value: f64,
+}
+
+#[derive(Debug, Clone)]
+pub struct HumiditySensorTemplate {
+    pub metadata: LocalisedSensorMetadata,
+    pub unit: String,
+}
+
+impl From<HumiditySensorTemplate> for HumiditySensor {
+    fn from(st: HumiditySensorTemplate) -> Self {
+        Self {
+            metadata: st.metadata,
+            unit: st.unit,
+            ..Default::default()
```

Comment:
I'm not sure if I'm happy about `default` being used for `From<XSensorTemplate>`. Wouldn't it be better if we'd use a custom `fn from(st: HumiditySensorTemplate, value: f64) -> Self` instead? It doesn't work in combination with `.into()`, but it ensures that the correct value is always set.

Reply:
Yes, this does seem a bit cleaner.